sdk/servicebus/Azure.Messaging.ServiceBus/tests/RuleManager/RuleManagerLiveTests.cs

```diff
@@ -0,0 +1,589 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus.Administration;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.RuleManager
+{
+    public class RuleManagerLiveTests : ServiceBusLiveTestBase
+    {
+        [Test]
+        public async Task AddGetAndRemoveRules()
+        {
+            await using (var scope = await ServiceBusScope.CreateWithTopic(enablePartitioning: false, enableSession: false))
+            {
+                await using var client = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString);
+
+                ServiceBusRuleManager ruleManager = client.CreateRuleManager(scope.TopicName, scope.SubscriptionNames.First());
+                var sqlRuleName = "sqlRule";
+                var correlationRuleName = "correlationRule";
+
+                var rules = (await ruleManager.GetRulesAsync()).ToList();
+                Assert.AreEqual(1, rules.Count());
+                var firstRule = rules[0];
+                Assert.AreEqual(RuleProperties.DefaultRuleName, firstRule.Name);
+                Assert.Null(firstRule.Action);
+
+                await ruleManager.AddRuleAsync(sqlRuleName, new SqlRuleFilter("price > 10"));
+
+                var ruleOptions = new CreateRuleOptions(correlationRuleName)
+                {
+                    Filter = new CorrelationRuleFilter
+                    {
+                        CorrelationId = "correlationId",
+                        Subject = "label",
+                        MessageId = "messageId",
+                        ApplicationProperties =
+                            {
+                                {"key1", "value1"}
+                            },
+                        ReplyTo = "replyTo",
+                        ReplyToSessionId = "replyToSessionId",
+                        SessionId = "sessionId",
+                        To = "to"
+                    },
+                    Action = new SqlRuleAction("Set CorrelationId = 'newValue'")
+                };
+                await ruleManager.AddRuleAsync(ruleOptions);
+
+                rules = (await ruleManager.GetRulesAsync()).ToList();
```

Comment:
`GetRulesAsync` returns a list by contract.   Do we need to `ToList()` here for some reason?